config/config-logging.yaml

```diff
@@ -19,36 +19,41 @@ metadata:
   namespace: knative-serving
   labels:
     serving.knative.dev/release: devel
-data:
+
```

Comment:
This configuration file is mainly for vendors and I don't think customers need to make any changes here or if we care to preserve their changes. Should we keep the original form of this?

Reply:
Discussed on chat.  Vendors can still modify these files in releases, if they want to specify values that users shouldn't override (and each `apply` will clobber any user changes).  If they want to let users configure them then they can omit them as we do here.